packages/lodestar/src/chain/stateTransition/epoch/finalUpdates.ts

```diff
@@ -5,12 +5,18 @@
 import {hashTreeRoot} from "@chainsafe/ssz";
 import BN from "bn.js";
 
-import {BeaconState, HistoricalBatch, ValidatorIndex} from "@chainsafe/eth2.0-types";
+import {BeaconState, HistoricalBatch} from "@chainsafe/eth2.0-types";
 import {IBeaconConfig} from "@chainsafe/eth2.0-config";
 
 import {bnMin, intDiv} from "../../../util/math";
 
-import {getActiveValidatorIndices, getCurrentEpoch, getRandaoMix, getShardDelta, getCompactCommitteesRoot} from "../util";
+import {
+  getActiveValidatorIndices,
+  getCompactCommitteesRoot,
+  getCurrentEpoch,
+  getRandaoMix,
+  getShardDelta
+} from "../util";
 
 
 export function processFinalUpdates(config: IBeaconConfig, state: BeaconState): void {
```

Comment:
You failed to merge:
https://github.com/ChainSafe/lodestar/pull/435/files#diff-487ce9f2b3677b91e7b8530221757b57R62

Reply:
its in there, I guess it was merged into interop before